lib/finders/ComponentFinder.js

```diff
@@ -231,6 +239,25 @@ class ComponentFinder extends events.EventEmitter {
 				this._removeComponent(component);
 				this.emit('unlink', component);
 			});
+
+		return this._readyWatchers;
+	}
+
+	/**
+	 * Close watchers
+	 *
+	 * @return {Promise} Promise resolved when all watcher will be close
+	 */
+	closeWatch() {
+		if (this._readyWatchers) {
+			const promise = this._readyWatchers;
+
+			this._readyWatchers = null;
+
+			return promise.then(watchers => watchers.map(watcher => watcher.close()));
```

Comment:
Why are you using `map` here? I bet the `close` method does not return a Promise here, right?